openpype/hosts/blender/api/render_lib.py

```diff
@@ -212,14 +213,13 @@ def imprint_render_settings(node, data):
 
 
 def prepare_rendering(asset_group):
-    name = asset_group.name
+    name = Path(asset_group.name)
```

Comment:
Is the asset group's name a Path? Is that correct?


Also you're turning it into `Path()` here just to do str(name)` later; is it a redundant conversion?

Reply:
You're right 😅 there is `get_render_product` that uses it as `Path`, and `set_node_tree` that uses it as `str`! I guess it's better to just initialize it as `Path` when passing it to `get_render_product`! I'll change it!